bsoid_app.py

```diff
@@ -1,3 +1,18 @@
+import os
+
+working_dir = os.environ.get('WORKING_DIR_BSOID', '')
+prefix = os.environ.get('PREFIX_BSOID', '')
+framerate = os.environ.get('FRAMERATE_BSOID', '')
```

Comment:
I'd do parsing stuff as `float` etc potentially here. So you don't have to do it inconsistently in the underlying classes

Reply:
Unless framerate should only be parsed to `float` in specific places